.travis.yml

```diff
@@ -1,49 +1,58 @@
 matrix:
-  include:
-    - os: osx
-      osx_image: xcode9.4
-      language: node_js
-      node_js: "10"
-      env:
-        - ELECTRON_CACHE=$HOME/.cache/electron
-        - ELECTRON_BUILDER_CACHE=$HOME/.cache/electron-builder
-      script:
-        # Build the release package.
-        # Draft a new release when a new version was pushed to master.
-        - yarn build --publish ${DEPLOY_TYPE}
-    - os: linux
-      services: docker
-      language: generic
-      script:
-        # Build the release package.
-        # Draft a new release when a new version was pushed to master.
-        - >
-          docker run --rm \
-            --env-file <(env | grep -iE 'DEBUG|NODE_|ELECTRON_|YARN_|NPM_|CI|CIRCLE|TRAVIS|APPVEYOR_|CSC_|_TOKEN|_KEY|AWS_|STRIP|BUILD_') \
-            -v ${PWD}:/project \
-            -v ~/.cache/electron:/root/.cache/electron \
-            -v ~/.cache/electron-builder:/root/.cache/electron-builder \
-            electronuserland/builder:wine \
-            /bin/bash -c "yarn build --linux --win --publish ${DEPLOY_TYPE}"
+    include:
+        - os: osx
+          osx_image: xcode9.4
+          language: node_js
+          node_js: "10"
+          env:
+              - ELECTRON_CACHE=$HOME/.cache/electron
+              - ELECTRON_BUILDER_CACHE=$HOME/.cache/electron-builder
+          script:
+              # Run mocha tests.
+              - yarn test
+          after_success:
+              # Build the release package.
+              # Draft a new release when a new version was pushed to master.
+              - yarn build --publish ${DEPLOY_TYPE}
+        - os: linux
+          services: docker
+          language: generic
+          script:
+              # Run mocha tests using a virtual screen.
+              - xvfb-run yarn test
+          after_success:
+              # Build the release package.
+              # Draft a new release when a new version was pushed to master.
+              - >
+                  docker run --rm \
+                    --env-file <(env | grep -iE
+                  'DEBUG|NODE_|ELECTRON_|YARN_|NPM_|CI|CIRCLE|TRAVIS|APPVEYOR_|CSC_|_TOKEN|_KEY|AWS_|STRIP|BUILD_')
+                  \
```

Comment:
:eyes: shouldn't this one be on the previous line?

Reply:
Yep, this was prettier splitting the lines as it was deemed too long. I've fixed it now. Good catch, thanks.